src/main/java/net/starlark/java/syntax/FlowStatement.java

```diff
@@ -21,7 +21,7 @@ public final class FlowStatement extends Statement {
 
   /** @param kind The label of the statement (break, continue, or pass) */
   FlowStatement(FileLocations locs, TokenKind kind, int offset) {
-    super(locs);
+    super(locs, Kind.FLOW);
     this.kind = kind;
```

Comment:
Add a comment flagging the fact that a private kind field exists at two levels of the class hierarchy? 
Or rename this field to token throughout?

Reply:
Good catch, I'll rename it to `flowKind` in the merge. (Instead of a comment, let's eliminate the need for one.)